rusty_pizza_server/src/order_model/user.rs

```diff
@@ -0,0 +1,33 @@
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct User {
+    name: String,
+}
+
+impl User {
+    pub fn new(name: String) -> User {
+        User { name }
+    }
+
+    pub fn set_name(&mut self, name: String) {
+        self.name = name
+    }
+
+    pub fn get_name(&self) -> String {
+        String::from(self.name.as_str())
```

Comment:
I'm asking myself what I would do with the name of the user. Most likely I would simply print it. For that reason a reference `&String` would be sufficient. In case a permanent instance is required I could still clone the value the reference points to. So I would change this to

```suggestion
    pub fn get_name(&self) -> &String {
        &self.name
    }
```

If you really want to copy the name you should use

```suggestion
        self.name.clone()
```